src/common/util.ts

```diff
@@ -91,7 +91,12 @@ export function printInfo(cmd: Command): void {
 export async function downloadFile(url: string, dest: string): Promise<void> {
   const destDir = path.dirname(dest);
   mkdirpSync(destDir); // ensure dir exists
-  await promisify(pipeline)((await fetch(url)).body, createWriteStream(dest));
+  const response = await fetch(url);
+  if (response.status === 200) {
```

Comment:
```suggestion
  if (response.status === 200 && response.body) {
```
I think its an error if there is nothing being written?